javascripts/background_scripts/first_run.js

```diff
@@ -64,21 +64,21 @@ var firstRun = {
     // Initialize the spoofing glyph
     // The generated string is not cryptographically secure and should not be used
     // for anything other than the glyph.
-    if (localStorage["glyph_cells"] === undefined) {
+    if (localStorage.glyph_cells === undefined) {
 
       // Dissable the posting button by default if the user already has
       // the extension installed.
-      if ( localStorage["posting_content_server_url"] !== undefined ) {
+      if ( localStorage.posting_content_server_url !== undefined ) {
         localStorage["Options:DissableButton"] = "true";
       }
 
-      localStorage["glyph_color"] = Math.floor(Math.random()*16777215).toString(16);
-
+      localStorage.glyph_color = Math.floor(Math.random()*16777215).toString(16);
+      var i;
       var glyph_cells = ((Math.random() < 0.5) ? "false" : "true");
       for(i = 0; i < 14; i++) {
```

Comment:
It would be more consistent to move the `var i;` into the `for` loop to be more consistent with the rest of the code base.